cli/wsproxy.ml

```diff
@@ -117,18 +115,18 @@ let handler sock msg =
     let localhost = Unix.inet_addr_loopback in
     let addr = Unix.ADDR_INET(localhost, int_of_string sport) in
     proxy sock addr protocol
-  | _ -> Lwt_log.warning "Malformed msg: not proxying"
+  | _ -> Logs_lwt.warn (fun m -> m "Malformed msg: not proxying")
 
 
-let _ = 
+let _ =
+  Logs.set_reporter (Logs_fmt.reporter ());
```

Comment:
Do we need a different reporter for Lwt?
As suggested by 
https://erratique.ch/software/logs/doc/Logs_lwt/index.html#report_ex and
https://erratique.ch/software/logs/doc/Logs/index.html#setupreporter

Reply:
We're debugging to `/dev/null` I don't think it matters if it's synchronous or no in this case.